drivers/sensor/bmi270/bmi270.c

```diff
@@ -552,14 +550,57 @@ static int bmi270_channel_get(const struct device *dev, enum sensor_channel chan
 	return 0;
 }
 
+#if defined(CONFIG_BMI270_TRIGGER)
+
+/* ANYMO_1.duration conversion is 20 ms / LSB */
+#define ANYMO_1_DURATION_MSEC_TO_LSB(_ms)	\
+	BMI270_ANYMO_1_DURATION(_ms / 20)
+
+static int bmi270_write_anymo_threshold(const struct device *dev,
+					struct sensor_value val)
+{
+	struct bmi270_data *data = dev->data;
+
+	/* this takes configuration in g. */
+	if (val.val1 > 0) {
+		LOG_DBG("anymo_threshold set to max");
+		val.val2 = 1e6;
+	}
+
+	/* max = BIT_MASK(10) = 1g => 0.49 mg/LSB */
+	uint16_t lsbs = (val.val2 * BMI270_ANYMO_2_THRESHOLD_MASK) / 1e6;
+	if (!lsbs) {
+		LOG_ERR("Threshold too low!");
+		return -EINVAL;
+	}
+
+	uint16_t anymo_2 = BMI270_ANYMO_2_THRESHOLD(lsbs)
+		| BMI270_ANYMO_2_OUT_CONF_BIT_6;
+
+	data->anymo_2 = anymo_2;
+	return 0;
+}
+
+static int bmi270_write_anymo_duration(const struct device *dev, uint32_t ms)
+{
+	struct bmi270_data *data = dev->data;
+	uint16_t val = ANYMO_1_DURATION_MSEC_TO_LSB(ms)
+		| BMI270_ANYMO_1_SELECT_XYZ;
+
+	data->anymo_1 = val;
+	return 0;
+}
+#endif /* CONFIG_BMI270_TRIGGER */
+
 static int bmi270_attr_set(const struct device *dev, enum sensor_channel chan,
 			   enum sensor_attribute attr, const struct sensor_value *val)
 {
 	int ret = -ENOTSUP;
 
-	if ((chan == SENSOR_CHAN_ACCEL_X) || (chan == SENSOR_CHAN_ACCEL_Y)
-	    || (chan == SENSOR_CHAN_ACCEL_Z)
-	    || (chan == SENSOR_CHAN_ACCEL_XYZ)) {
+	if ((chan == SENSOR_CHAN_ACCEL_X) ||
+	    (chan == SENSOR_CHAN_ACCEL_Y) ||
+	    (chan == SENSOR_CHAN_ACCEL_Z) ||
+	    (chan == SENSOR_CHAN_ACCEL_XYZ)) {
```

Comment:
This change isn't related to adding triggers and should be made in a separate commit

Reply:
I have changed this and verified that I've pushed it, but it seems as if something is configured to ignore "whitespace only" changes and I don't think it's locally. Would you mind either a) letting it slide, or b) removing the added newline yourself?

Reply:
Please check again. I don't object to the change, but it needs to be in a separate commit so it doesn't add noise to functional changes.